README.md

```diff
@@ -26,3 +26,14 @@ Then reference the rules in your `tslint.json` and enable the rules you want:
   "expect-length": true
 }
 ```
+
+## Fixable
+
+The rules `no-focused-tests` and `no-disabled-tests` can have a parameter `fixable` to fix them:
+
+```json
+{
+  "no-focused-tests": [true, "fixable"],
```

Comment:
What's the reason for adding the extra parameter? Is there ever a time when you would want to configure the rule not to be fixable?

Reply:
I've removed 'fixable' and just always including the fix in the rule. Published as version 1.5.0

Reply:
I would like to disable the rules during development and test to keep the `fdescribe` by example. But for the production build I want it to be fix.

Reply:
I don’t understand what you mean. I haven’t seen any other rules that are fixable that are configurable like that.

Reply:
Indeed I do't fix the fixable attribute is interesting for the other rules. But for the test, in development, i like to use fdescribe / fit. But not in prod...
I will use your fixable rules in dev with [tslint-defocus](https://github.com/Sergiioo/tslint-defocus) in prod.

Reply:
Still a bit confused. I also like to use fdescribe/fit in development but that still works with these rules. Just don't apply the fix while developing. And if you accidentally commit an fit/fdescribe your prod build process will just fail. Or are you using something that automatically fixes fixable rules?